dev-utils/win_installer/_base.sh

```diff
@@ -203,7 +206,7 @@ function cleanup_after {
     rm -Rf "${MINGW_ROOT}"/libexec
     rm -Rf "${MINGW_ROOT}"/share/gtk-doc
     rm -Rf "${MINGW_ROOT}"/include
-    rm -Rf "${MINGW_ROOT}"/var
+    rm -Rf "${MINGW_ROOT:?}"/var
```

Comment:
Do you know why it does the error checking only in this line and not the 50 others?

Reply:
Yes it "knows" about certain top-level directories that would be a disaster to remove and `/var` is one of them. I guess we could add this everywhere. 
Though might be easier to `test` for it earlier and exit if empty?

Reply:
I see, not sure..